agent/rpc_from.go

```diff
@@ -98,6 +98,12 @@ func (a *orbAgent) handleDatasetRemoval(rpc fleet.DatasetRemovedRPCPayload) {
 	a.removeDatasetFromPolicy(rpc.DatasetID, rpc.PolicyID)
 }
 
+func (a *orbAgent) handleAgentReset(payload fleet.AgentResetRPCPayload) {
+	// TODO implement logic for agent and backend reset
```

Comment:
we can remove the comments? and pass in the `payload` here so `Restart` has access to its contents. it can check FullReset there for now.